### ISSUE TITLE
Unable to build from both source and AUR 

### ISSUE THREAD

**Issue by xi-pinmping** [fluix-dev/sway-borders#70]
So I updated an old install and forgot to ignore wlr and sway and since they are from git I have no old packages to go back to. I thought everything was fine until I tried to boot in and it failed. This prompted a reinstall and it failed to build from the AUR. I have tried to also go from source as described in other issue guides but I have had no success compiling yet. Is there a specific version I should be using somewhere? I have cloned the current gitlab repo for wlroots and the newest from sway-borders. Thanks! Build below..

''ninja -C build  
ninja: Entering directory `build'
[350/571] Compiling C object sway/sway.p/ipc-json.c.o
FAILED: sway/sway.p/ipc-json.c.o 
cc -Isway/sway.p -Isway -I../sway -Iinclude -I../include -Isubprojects/wlroots/include -I../subprojects/wlroots/include -Iprotocols -I/usr/include/cairo -I/usr/include/lzo -I/usr/include/libpng16 -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/json-c -I/usr/include/libevdev-1.0 -I/usr/include/pango-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -O0 -g -DWLR_USE_UNSTABLE -Wno-unused-parameter -Wno-unused-result -Wno-missing-braces -Wundef -Wvla '-DSYSCONFDIR="//usr/local/etc"' '-DSWAY_VERSION="1.8-dev-3890d2a2 (" __DATE__ ", branch '"'"'master'"'"')"' -fmacro-prefix-map=../= -pthread -MD -MQ sway/sway.p/ipc-json.c.o -MF sway/sway.p/ipc-json.c.o.d -o sway/sway.p/ipc-json.c.o -c ../sway/ipc-json.c
../sway/ipc-json.c: In function ‘ipc_json_output_adaptive_sync_status_description’:
../sway/ipc-json.c:115:14: error: ‘WLR_OUTPUT_ADAPTIVE_SYNC_UNKNOWN’ undeclared (first use in this function); did you mean ‘WLR_OUTPUT_ADAPTIVE_SYNC_ENABLED’?
  115 |         case WLR_OUTPUT_ADAPTIVE_SYNC_UNKNOWN:
      |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |              WLR_OUTPUT_ADAPTIVE_SYNC_ENABLED
../sway/ipc-json.c:115:14: note: each undeclared identifier is reported only once for each function it appears in
[352/571] Compiling C object sway/sway.p/lock.c.o
ninja: build stopped: subcommand failed.
`

**Comment by Lexasovsky** [comment]
I have the same

**Comment by xi-pinmping** [comment]
I was eventually able to clone a commit of the wlroots repo from a date slightly before that of the last sway-borders update (away from the device right now but when I am able to I’ll update with the specific commit I used) and was able to build it and can get to the environment by executing the sway build executable in the build directory but however I can not execute sway as a command and get the environment. It tells me libwlroots.so.11 is unavailable. I have tried ninja -C build install and appending doas to the previous command. Doas executes without issue but I still am not able to boot into sway even after reboot. I’ve tried copying executable into my /usr/local/bin or replacing the file there with a link to the build executable but that provides the same error as before. I’m lost entirely lol. I also tried changing my sway entry to use the compiled executable but that doesn’t work either. All the same error.
